Makefile

```diff
@@ -67,6 +67,10 @@ build/bin/$(BIN)-windows-amd64:
 		golang:$(GOVERSION) go build -a -installsuffix cgo -o build/bin/$(BIN)-windows-amd64 \
 		-ldflags "-X 'github.com/giantswarm/gsctl/config.Version=$(VERSION)' -X 'github.com/giantswarm/gsctl/config.BuildDate=$(BUILDDATE)' -X 'github.com/giantswarm/gsctl/config.Commit=$(COMMIT)'"
 
+# run unittests
+gotest:
+	go test -cover ./util ./config
```

Comment:
I'd used $(glide novendor) here rather than specifying the packages. There is also a change to Golang in progress to not run vendor tests which is a better default.

https://go-review.googlesource.com/c/38745/